apps/ledger-live-mobile/src/screens/Platform/exchange/CompleteExchange.tsx

```diff
@@ -13,6 +13,7 @@ import { mevProtectionSelector } from "~/reducers/settings";
 import { SignedOperation } from "@ledgerhq/types-live";
 import { Transaction } from "@ledgerhq/live-common/generated/types";
 import { HOOKS_TRACKING_LOCATIONS } from "~/analytics/hooks/variables";
+import LoadingIndicator from "~/newArch/features/Web3Hub/components/ManifestsList/LoadingIndicator";
```

Comment:
I think we use this import now with [the newArch](https://ledgerhq.atlassian.net/wiki/spaces/WXP/pages/4508058220/Code+Guidelines+-+React+Native)
```suggestion
import LoadingIndicator from "LLD/features/Web3Hub/components/ManifestsList/LoadingIndicator";
```